decidim-proposals/db/migrate/20230427105700_change_state_on_collaborative_drafts.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+class ChangeStateOnCollaborativeDrafts < ActiveRecord::Migration[6.1]
```

Comment:
I think it's better to be consistent with the name of these migrations, as they're all pretty similar, but some are "ChangeStatus" and others are "ChangeState":
```
20230515092519_change_status_on_amendments.decidim.rb
20230515092520_change_status_on_reminder_records.decidim.rb
20230515092521_change_state_on_collaborative_drafts.decidim_proposals.rb
20230515092522_change_state_on_proposals.decidim_proposals.rb
```
How about changing it all to `ChangeStates` (as this is the constant's name in all the classes)?
(You'd need to also change the filename of course)

```suggestion
class ChangeStatesOnCollaborativeDrafts < ActiveRecord::Migration[6.1]
```